app/experimenter/nimbus-ui/src/components/PageRequestReview/index.stories.tsx

```diff
@@ -53,267 +44,73 @@ storiesOf("pages/RequestReview", module)
     </RouterSlugProvider>
   ));
 
-storiesOf("pages/RequestReview/EXP-1055", module)
+storiesOf("pages/RequestReview/EXP-1143", module)
```

Comment:
I know this description is temporary, but it could be nice in the meantime to call this something like, `EXP-1143; end experiment review flow` as well as a description for `EXP-1143, launch experiment review flow` just to make it more explicit/clear. I went back to reference [this comment](https://github.com/mozilla/experimenter/pull/4973#issuecomment-812201684) a couple times (thanks for those handy links BTW!)